package/package.py

```diff
@@ -46,9 +48,10 @@ def spec_file_name(self, ext: str):
             return "mount-s3-suse.spec"
         return "mount-s3.spec"
 
-
-OPT_PATH = "opt/aws/mountpoint-s3"
-
+@dataclass
+class MountpointArtifact:
+    binary_path: str
```

Comment:
Perhaps using [`Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path) objects would be better here - it would allow us to drop the use of `os.path.join` all over the place, and the type makes more sense here.

Reply:
I agree, but all the code uses strings and low-level `os.path` module at the moment, I'd rather to have a separate refactoring PR to address that for the whole code